1_fetch.R

```diff
@@ -41,7 +42,30 @@ p1_targets_list <- list(
     p1_nhd_reaches,
     download_nhdplus_flowlines(p1_drb_comids_all_tribs$COMID)
   ),
-
+  
+  ## May take a while to run
+  tar_target(
+    p1_nhd_catchments,
+    get_nhdplusv2_catchments(comid = p1_nhd_reaches$comid)
+  ),
+  
+  tar_target(
+    p1_nhd_catchments_along_nhm,
+    get_nhdplusv2_catchments(comid = p1_nhd_reaches_along_NHM$comid)
+  ),
+  
+  tar_target(
+    p1_nhm_catchments_upstream,
+    {sf_use_s2(FALSE)
```

Comment:
I'm curious whether there was specific behavior you noticed that led to using `sf_use_s2(FALSE)` here? Also, this target results in the following message getting printed to the console for each segidnat group: `although coordinates are longitude/latitude, st_union assumes that they are planar`. Since this message obscures other pipeline messages, perhaps we should either suppress this message or transform the polygons to a projected CRS (like 5070)? What do you think?